python.md

```diff
@@ -65,6 +65,13 @@ Python relies on native code to achieve high performance.  For scientific and
 numerical applications NumPy and SciPy provide an interface to high performance
 computing libraries such as ATLAS, BLAS, BLIS, OpenBLAS, etc.  These libraries
 contain code tuned for Graviton processors.
+It is recommended to use the latest software versions as much as possible. If the latest
+version migration is not feasible, please ensure that it is atleast the minimum version recommended
+below. This will guarantee stable and performant libraries for Graviton processors.
```

Comment:
please put the why. There was a bug in OpneBLAS which was fixed in x.y.z and this version was used in Scipi/numpy y.z....